### DISCUSSION TITLE
Can't built due to Site.IsMultiLingual and can't find it.

### DISCUSSION THREAD

**Discussion by sergedoub** [thuliteio/doks#1302] [Q&A]
Tried everything to find what's causing this but can't figure it out. 

`hugo v0.136.2+extended darwin/arm64 BuildDate=2024-10-17T14:30:05Z VendorInfo=brew

ERROR deprecated: .Site.IsMultiLingual was deprecated in Hugo v0.124.0 and will be removed in Hugo 0.137.0. Use hugo.IsMultilingual instead.
Built in 368 ms`

**Comment by h-enk** [comment]
https://github.com/thuliteio/doks-core/issues/104

**Reply by h-enk** [reply]
Fixed with [@thulite/doks-core v1.8.0](https://github.com/thuliteio/doks-core/releases/tag/v1.8.0) and [create-thulite v1.7.3](https://github.com/thuliteio/create-thulite/releases/tag/v1.7.3)
